packages/mui-material/src/MenuList/MenuList.test.js

```diff
@@ -63,7 +63,7 @@ describe('<MenuList />', () => {
   });
 
   describe('actions: adjustStyleForScrollbar', () => {
-    const expectedPadding = `${getScrollbarSize(document)}px`;
+    const expectedPadding = `${getScrollbarSize(document.defaultView)}px`;
```

Comment:
I believe for those tests, the following could be in order:
```suggestion
    const expectedPadding = `${getScrollbarSize(window)}px`;
```

Reply:
Oh, didn't see it at first, but makes sense. It would also remove the need of checking for null in the function. I'll make a commit